docs/knowledge_base/index.md

```diff
@@ -78,7 +78,7 @@ Learn how to use Taipy and improve your skills.
 # Tutorials
 
 <ul class="tp-cards-list">
-  <li>
+  <li keywords="gui vizelement input text button chart navbar table layout part expandable menu state interactive multi-page callback on_change markdown">
```

Comment:
`keywords` alone is note a valid HTML attribute. You must prefix custom attributes with `data-`. So it would be `data-keywords` instead.